cvmfs/options.cc

```diff
@@ -74,10 +53,41 @@ static string EscapeShell(const std::string &raw) {
 }
 
 
-void ParsePath(const string &config_file, const bool external) {
-  LogCvmfs(kLogCvmfs, kLogDebug, "Parsing config file %s", config_file.c_str());
+void FastOptionsManager::ParsePath(const string &config_file,
+                                 const bool external __attribute__((unused))) {
```

Comment:
A better style for multi-line signatures (only for future reference):

    void FastOptionsManager::ParsePath(
      const string &config_file,
      const bool external __attribute__((unused))
    ) {